hocon/runtime/src/main/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigSourceProvider

```diff
@@ -0,0 +1,2 @@
+io.quarkus.config.hocon.runtime.ApplicationHoconConfigSourceLoader$InFileSystem
+io.quarkus.config.hocon.runtime.ApplicationHoconConfigSourceLoader$InClassPath
```

Comment:
With the builder approach, the ServiceLoader registration is no longer required since we register the sources directly with the builder.